app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntryUpdate.kt

```diff
@@ -4,15 +4,26 @@ import com.spotify.mobius.Next
 import com.spotify.mobius.Next.noChange
 import com.spotify.mobius.Update
 import org.simple.clinic.mobius.justEffect
+import org.simple.clinic.mobius.justEffects
 
 class BloodPressureEntryUpdate : Update<BloodPressureEntryModel, BloodPressureEntryEvent, BloodPressureEntryEffect> {
   override fun update(
       model: BloodPressureEntryModel,
       event: BloodPressureEntryEvent
   ): Next<BloodPressureEntryModel, BloodPressureEntryEffect> {
     return when (event) {
-      is SystolicChanged -> justEffect(HideBpErrorMessage)
+      is SystolicChanged -> if (isSystolicValueComplete(event.systolic)) {
+        justEffects(HideBpErrorMessage, ChangeFocusToDiastolic)
+      } else {
+        justEffect<BloodPressureEntryModel, BloodPressureEntryEffect>(HideBpErrorMessage)
```

Comment:
The signature of `justEffects` also supports sending a single effect. Should we use that instead of this single parameter method?

Reply:
I prefer to have two different functions to reveal more information. I am open to having a single function as long as it doesn't sound ambiguous. However I have changed `justEffect` to internally call `justEffects`.

Reply:
Or we could use a function named `dispatch` that can do 1 or more effects. WDYT?

Reply:
Looks good to me, let's see in the other PR.